monasca-aggregator/Dockerfile

```diff
@@ -1,27 +1,47 @@
-from alpine:3.7
+from golang:1.10.3-alpine3.7 as builder
 
 arg AGGREGATOR_REPO=https://github.com/monasca/monasca-aggregator
 arg AGGREGATOR_BRANCH=master
 
 env GOPATH=/go
 
-run apk add --no-cache openssl py2-jinja2 librdkafka && \
-  apk add --no-cache --virtual build-dep \
-    librdkafka-dev git go glide make g++ openssl-dev && \
-  mkdir -p $GOPATH/src/github.com/monasca/monasca-aggregator && \
+run apk add --no-cache openssl py2-jinja2 && \
+  apk add --no-cache --virtual build-dep git go glide make g++ openssl-dev
+
+# librdkafka is not available in a stable alpine release
+run apk add --update --no-cache \
+        libressl \
+    --repository http://nl.alpinelinux.org/alpine/edge/main  \
```

Comment:
Use one `ARG` for `http://nl.alpinelinux.org/alpine` so it could be changed in one place only.
Also looking at https://mirrors.alpinelinux.org/ I think that `uk.alpinelinux.org` would be faster (but `nl` is also fine with me).

Reply:
agreed, will add. I have no preference on which mirror it uses, will switch to uk

Reply:
Well, you replaced http://nl.alpinelinux.org/alpine/edge/main with `ALPINE_EDGE_REPO` in only one place. What about here and line 48?